src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -510,6 +511,10 @@
 						"	set the fake max players count to (online players count + 1)",
 						"	set the shown icon to a random server icon out of {server-icons::*}")
 				.since("2.3");
+		Skript.registerEvent("Unknown Command", SimpleEvent.class, UnknownCommandEvent.class, "unknown command")
```

Comment:
You need to use a classExists here as well.
```
		if (Skript.classExists("com.destroystokyo.paper.event.player.PlayerArmorChangeEvent")) {
			Skript.registerEvent("Armor Change", SimpleEvent.class, PlayerArmorChangeEvent.class, "[player] armor change[d]")
				.description("Called when armor pieces of a player are changed.")
				.requiredPlugins("Paper")
				.examples("on armor change:",
					"	send \"You equipped %event-item%!\"")
				.since("2.5");
		}
```
Remember to add ``.requiredPlugins("Paper")`` too